performance-tuning-methods.md

```diff
@@ -0,0 +1,281 @@
+    >
+    > - The execute time may also be significantly greater than the total time of the KV request plus the tso_wait time. This means that the SQL execution time is spent mostly inside the TiDB executor. Here are two common examples:
+    >
+        > - Example 1: After TiDB executor reads a large amount of data from TiKV, it needs to do complex association and aggregation inside TiDB, which consumes a lot of time.
+        > - Example 2: The application experiences serious write statement lock conflicts. Frequent lock retries result in long `Retried execution time`.
+
+**Example 2: OLTP read intensive load**
+
+![OLTP](/media/performance/oltp_normal_db_time.png)
+
+- Database Time by SQL Type: Major time-consuming statements are `SELECT`, `COMMIT`, `UPDATE`, and `INSERT`, among which `SELECT` consumes most database time.
+- Database Time by SQL Phase: Most time is consumed in the execute phase in green.
+- SQL Execute Time Overview: In SQL execution phase, pd tso_wait in dark brown, KV Get in blue, and Prewrite and Commit in green are time-consuming.
+
+**Example 3: Read-only OLTP load**
+
+![OLTP](/media/performance/oltp_long_compile_db_time.png)
+
+- Database Time by SQL Type: Mainly are `SELECT` statements.
+- Database Time by SQL Phase: Major time-consuming phases are compile in orange and execute in green. Latency in the compile phase is the highest, indicating that TiDB is taking too long to generate execution plans and the root cause needs to be further determined based on the subsequent performance data.
+- SQL Execute Time Overview: The KV BatchGet requests in blue consume the most time during SQL execution.
+
+> **Note:**
+>
+> In example 3, `SELECT` statements need to read thousands of rows concurrently from multiple TiKVs. Therefore, the total time of the BatchGet request is much longer than the execution time.
+
+**Example 4: Lock contention load**
+
+![OLTP](/media/performance/oltp_lock_contention_db_time.png)
+
+- Database Time by SQL Type: Mainly are `UPDATE` statements.
+- Database Time by SQL Phase: Most time is consumed in the execute phase in green.
+- SQL Execute Time Overview: The KV request PessimisticLock shown in red consumes the most time during SQL execution, and the execution time is obviously longer than the total time of KV requests. This is caused by serious lock conflicts in write statements and frequent lock retries prolong `Retried execution time`. Currently, TiDB does not measure `Retried execution time`.
+
+### TiDB key metrics and cluster resource utilization
+
+#### Query Per Second, Command Per Second, and Prepared-Plan-Cache
+
+The following three panels in Performance Overview present the application load profile, how the application interacts with TiDB, and whether the application fully utilizes TiDB's [execution plan cache](/sql-prepared-plan-cache.md).
```

Comment:
```suggestion
By checking the following three panels in Performance Overview, you can learn the application workload type, how the application interacts with TiDB, and whether the application fully utilizes TiDB [execution plan cache](/sql-prepared-plan-cache.md).
```

Reply:
reason: the three panels do not present such info directly so users need to check and decide that accordingly.